src/Monitor/Monitor/help/New-AzActivityLogAlertCondition.md

```diff
@@ -26,6 +26,8 @@ The **New-AzActivityLogAlertCondition** cmdlet creates new activity log alert co
 ### Example 1: Create a new activity log alert condition object in memory.
 ```
 PS C:\>$condition = New-AzActivityLogAlertCondition -Field "Requests" -Equal "OtherField"
+PS C:\>$condition.Field
```

Comment:
@theheatDK would you mind including the corresponding output for each of these properties? This will give users an idea of what is returned when accessing these properties

Reply:
@cormacpayne How does this look?

Reply:
LGTM :shipit: